app/views/components/page.scala.html

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  *@
 
-@import models.ui.{Page, Paragraph, HyperLink, H1, H2, H3, BulletPointList, Question}
+@import models.ui.{Page, Paragraph, Link, H1, H2, H3, BulletPointList, Question}
```

Comment:
Link is not actually used in this component.

Reply:
will remove